conda_press/condatools.xsh

```diff
@@ -413,10 +413,20 @@ class ArtifactInfo:
         self.meta_yaml = None
         self.files = None
         self.artifactdir = artifactdir
+        self._exclude_deps = []
+        self.exclude_deps = exclude_deps
```

Comment:
This line should be removed since it is a property below.  These lines should probably be just

```python
self._exclude_deps = exclude_deps
```

Reply:
Sure thing! I forgot to remove it! Thanks!

Reply:
I think it would be good to preserve the property just because there is a small logic in there, but the part which I am setting the private attribute directly can be removed indeed.